api/src/main/java/org/ehcache/spi/serialization/Serializer.java

```diff
@@ -28,8 +28,9 @@
  * <dl>
  *   <dt><code><i>Serializer</i>(ClassLoader loader)</code>
  *   <dd>This constructor is used to initialize the serializer for transient caches.
```

Comment:
No, it is the constructor we use when no persistent tier is configured. Which is what makes this implicit constructor contract even more weird. A stateful serializer still needs to have this constructor if it is fit for transient caches.